### PR TITLE
Fix broken link in README

### PR DESCRIPTION
**Pull request by ctrueden**
The napari docs are now versioned.

### CODE REVIEW

**Comment by tlambert03** [comment]
thanks for the fix @ctrueden,  I'm going to use the `stable` link though: https://napari.org/guides/stable/rendering.html

File: README.md [hunk 1]
```diff
@@ -105,7 +105,8 @@ pip install napari-omero
 - remote loading can be very slow still... though this is not strictly an issue
   of this plugin.  Datasets are wrapped as delayed dask stacks, and remote data
   fetching time can be significant.  Plans for [asynchronous
-  rendering](https://napari.org/docs/explanations/rendering.html) in napari and
+  rendering](https://napari.org/docs/0.4.7/explanations/rendering.html) in
```
Comment (tlambert03) [inline README.md:108]:
```suggestion
  rendering](https://napari.org/guides/stable/rendering.html) in
```

**Review by tlambert03 — APPROVED** [review]

**Comment by ctrueden** [comment]
> I'm going to use the stable link though

Ahh, I looked for a stable link (I tried `https://napari.org/docs/stable/explanations/rendering.html` and similar) but couldn't find it. I'm glad one exists. :+1:

**Comment by tlambert03** [comment]
yeah, it's a bit of a mess 🥴  ... thanks again!

**Pull request merged**